overlay/mruby-builder/default.nix

```diff
@@ -1,6 +0,0 @@
-{
-  pkgs' ? (import <nixpkgs> {                      
-    overlays = [(import ../builder/overlay.nix)];  
-  })               
-}: pkgs'
-
```

Comment:
This file really wasn't needed. It was only useful when I first started working on the mruby stuff, which was eons ago :). Removing it removes one slim layer of complexity.